www/pfSense-pkg-squid/files/usr/local/pkg/squid.inc

```diff
@@ -2095,7 +2084,7 @@ function squid_generate_rules($type) {
 				}
 				/* Handle PPPOE case */
 				if (($config['pppoe']['mode'] == "server" && $config['pppoe']['localip']) || (function_exists("is_pppoe_server_enabled") && is_pppoe_server_enabled())) {
-					$rules .= "no rdr on $PPPOE_ALIAS proto tcp from any to { 192.168.0.0/16, 172.16.0.0/12, 10.0.0.0/8 } port {$pf_rule_port}\n";
+					$rules .= "no rdr on $pppoe proto tcp from any to { 192.168.0.0/16, 172.16.0.0/12, 10.0.0.0/8 } port {$pf_nat_ports}\n";
```

Comment:
@rbgarga - the changes here were done as suggested by @jim-p (https://github.com/pfsense/FreeBSD-ports/pull/289#discussion_r99846097) - as I've noted, I don't have any such setup to test, and generally this is something completely outside of my knowledge.